src/reducers/loginReducer.js

```diff
@@ -0,0 +1,16 @@
+import { LOGIN, SIGN_UP } from '../actions/loginActions'
+
+const initialState = {
+    isLoggedIn: false
+}
+
+export default function loginReducer(state = initialState, action) {
+    switch (action.type) {
+        case LOGIN:
+            return { isLoggedIn: true };
```

Comment:
Please read about immutability with redux and decide if this code still should be used this way.